Pear/Controller/SchedulingTimeViewController.swift

```diff
@@ -112,15 +112,19 @@ class SchedulingTimeViewController: UIViewController {
     private var backButton = UIButton()
     private var dayCollectionView: UICollectionView!
     private let dayLabel = UILabel()
-    private var errorBlurEffect: UIBlurEffect!
-    private var errorMessageAlertView: MessageAlertView!
-    private var errorMessageVisualEffectView: UIVisualEffectView!
     private let infoLabel = UILabel()
     private let nextButton = UIButton()
     private let noTimesWorkButton = UIButton()
     private var timeCollectionView: UICollectionView!
     private var timeScrollView: FadeWrapperView<UIScrollView>!
     private let titleLabel = UILabel()
+    
+    private var errorMessageAlertView: MessageAlertView!
+    private var errorMessageVisualEffectView: UIVisualEffectView!
+    
+    private var emailMessageAlertView: MessageAlertView!
+    private var emailMessageVisualEffectView: UIVisualEffectView!
```

Comment:
I think you can refactor so that these aren't `!`. It is safer that way and more consistent, but if it overcomplicates things in your opinion then it's ok to leave it.